conans/client/conan_api.py

```diff
@@ -153,7 +153,11 @@ def inspect(self, path, attributes, remote_name=None):
         for attribute in attributes:
             try:
                 attr = getattr(conanfile, attribute)
-                result[attribute] = attr
+                if attribute == "options":
+                    result[attribute] = {k: v._possible_values
+                                         for k, v in attr._package_options._data.items()}
```

Comment:
bit dirty. Could we have a method or property to get the items directly for representation?

Reply:
Fixed